### PR TITLE
Added ESS surface conductiivity (V2) polling

### PR DESCRIPTION
**Pull request by gordonparikh**
Adds ESS surface conductivity V2 polling, used by NDDOT for reporting friction readings.

### CODE REVIEW

**Comment by DougLau** [comment]
This query should happen in the QueryPavementRowV2 phase, since it's for a V2 object.  There is even a comment at line line 331 mentioning this.

**Comment by gordonparikh** [comment]
I saw the comment, but I didn't want to tie it up with the ice_or_water_depth polling since our sensors aren't reporting that. From what I could tell, it seemed preferred to add a new polling class/method to deal with the error handling rather than having multiple try/catch blocks in a single poll (I couldn't find any instances of that in the code). I can move it to QueryPavementRowV2 though if that is preferable.

**Comment by DougLau** [comment]
In QueryPavementRow, essSurfaceConductivityV2 will return NoSuchName for older V1 stations, making it so that none of that data is collected.

If your sensors return NoSuchName for essSurfaceIceOrWaterDepth, we'll have to make a new phase -- but if they just return 65,535 we can get by with just QueryPavementRowV2.

File: src/us/mn/state/dot/tms/server/WeatherSensorImpl.java [hunk 1]
```diff
@@ -644,6 +644,23 @@ public void setSubSurfTempNotify(Integer v) {
...
+	private transient Integer surface_conductivity_v2;
+
+	/** Get surface conductivity (V2) (null for missing) */
+	@Override
+	public Integer getSurfCondV2() {
+		return surface_conductivity_v2;
+	}
+
+	/** Set surface conductivity (V2) (null for missing) */
+	public void getSurfCondV2Notify(Integer v) {
```
Comment (DougLau) [inline src/us/mn/state/dot/tms/server/WeatherSensorImpl.java:657]: -> setSurfCondV2Notify

Comment (gordonparikh) [reply src/us/mn/state/dot/tms/server/WeatherSensorImpl.java:657]: Thanks for catching that

**Pull request closed without merging**